packages/suite-desktop/src-electron/index.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference path="../../suite/global.d.ts" />
```

Comment:
Why do we include this via a triple-slash directive and not, for example, by the [`files`](https://www.typescriptlang.org/tsconfig#files) field in tsconfig.json?

Reply:
What's the benefit? I prefer to handle references in code rather than in config.

Reply:
To me it looks 'cleaner' if it would be placed in `tsconfig.json` but I don't have a strong opinion on this.

Reply:
> What's the benefit?

Not a particularly strong one but I'd expect that using the tsconfig file is the canonical way to include files into the compilation. However I'm assuming that only because this is the first time I see a triple slash directive.

> I prefer to handle references in code rather than in config.

Mee too. However one usually uses the `require`/`import` JS directives rather than a *compiler directive*. If there isn't a dependency on the typescript level why are we linking them together on the compiler level? Maybe just a brief comment explaining why the file is included at all would be helpful. .)

Reply:
Comment added: 96c870e

Reply:
Seeing the comment I still don't understand why are we doing this. 😕 If some parts of `@suite` depend on `global.d.ts`, shouldn't they reference it the typescript way (`import`/`require`)? I also don't understand the "can" word. If they rely on `global.d.ts` and it's not available then compilation should fail. If they don't, `global.d.ts` shouldn't need to be included.

Also a meta comment: I think threads on a PR should be resolved by the OP rather than the PR author just as described in our [review process](https://docs.trezor.io/trezor-firmware/misc/review.html).